android/playground/src/main/java/com/kiwi/mobile/rnandroidplayground/AncillaryFactoryFragment.kt

```diff
@@ -0,0 +1,10 @@
+package com.kiwi.mobile.rnandroidplayground
+
+import com.facebook.react.ReactNativeHost
+import com.kiwi.rnkiwimobile.ancillaries.RNAncillaryFactoryFragment
+import com.kiwi.rnkiwimobile.ancillaries.RNAncillaryFactoryInitialProperties
+
+
+class AncillaryFactoryFragment: RNAncillaryFactoryFragment(RNAncillaryFactoryInitialProperties(service = "fast_track", bookingId = 123, kwAuthToken = "mock")) {
```

Comment:
Is this line too long?

Reply:
it is. Not sure what is the proper way to make it shorter. But I tried making it shorter